packages/snapshot/src/port/state.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import type { OptionsReceived as PrettyFormatOptions } from '@vitest/pretty-format'
-import type { ParsedStack } from '../../../utils/src/index'
+import type { ParsedStack } from '@vitest/utils'
```

Comment:
This is the only source code change. This change allows `unplugin-isolated-decl` to externalize this type import and then `rollup-plugin-dts` to bundle `@vitest/utils` types inside `@vitest/snapshot`.

Reply:
Checked the generated output types, looks good. 👍